lib/northbound_grpc.cpp

```diff
@@ -1282,12 +1285,14 @@ static void *grpc_pthread_start(void *arg)
 	/* Process inbound RPCs */
 	while (true) {
 		void *tag;
-		bool ok;
+		bool ok, res;
+
+		res = s_cq->Next(&tag, &ok);
+		if (!res || !ok)
```

Comment:
Let's not bother with the extra `res` value here. Most of the GRPC examples do not check that value in this use case so for someone comparing code it will be less confusing. The API indicates `ok` will always be false if the result is false, so let's just check for `!ok`.